sqlglot/dialects/trino.py

```diff
@@ -2,13 +2,35 @@
 
 from sqlglot import exp
 from sqlglot.dialects.presto import Presto
+from sqlglot.helper import seq_get
 
 
 class Trino(Presto):
+    class Parser(Presto.Parser):
+        FUNCTIONS = {
+            **Presto.Parser.FUNCTIONS,  # type: ignore
+            "DAY_OF_WEEK": lambda args: exp.DateOfWeek(
```

Comment:
let's remtove these overrides and just name it exp.DayOfWeek and have _sql_names["DAY_OF_WEEK", "DAYOFWEEK"]

Reply:
these should not have been added to trino, but to the base class presto because they are valid there

https://prestodb.io/docs/current/functions/datetime.html